src/pages/e-ticket.tsx

```diff
@@ -1,32 +1,28 @@
-import Image from 'next/image';
+import QRCode from "react-qr-code";
 import Link from 'next/link';
+import Image from 'next/image';
 
-// using https://github.com/dephraiim/qrcode
-// example: https://qr-service.vercel.app/api/code?text=https://freshmen2023.sgcu.in.th/&size=200
+// using https://www.npmjs.com/package/react-qr-code 
 
-const ETicket = () => {
+const ETicket = (qrcode: string) => {
```

Comment:
This component is a whole page and cannot accept props.
It should be `const qrcode = 'some mock value'`